src/send/SendEnterAmount.tsx

```diff
@@ -313,18 +330,7 @@ function SendEnterAmount({ route }: Props) {
                     : undefined
                 }
               />
-              <Touchable
-                borderRadius={TOKEN_SELECTOR_BORDER_RADIUS}
-                onPress={onTokenPickerSelect}
-                style={styles.tokenSelectButton}
-                testID="SendEnterAmount/TokenSelect"
-              >
-                <>
-                  <FastImage source={{ uri: token.imageUrl }} style={styles.tokenImage} />
-                  <Text style={styles.tokenName}>{token.symbol}</Text>
-                  <DownArrowIcon color={Colors.gray5} />
-                </>
-              </Touchable>
+              {tokenSelectorComponent}
```

Comment:
nit: you can inline this instead of using the extra `tokenSelectorComponent` variable